packages/runtime/runtime-definitions/api-report/runtime-definitions.legacy.alpha.api.md

```diff
@@ -233,7 +234,7 @@ export interface IFluidParentContext extends IProvideFluidHandleContext, Partial
     readonly isReadOnly?: () => boolean;
     readonly loadingGroupId?: string;
     makeLocallyVisible(): void;
-    readonly minVersionForCollab?: MinimumVersionForCollab;
+    readonly minVersionForCollab: MinimumVersionForCollab;
```

Comment:
(Too bad the property name and type name are spelled differently. Oh well, not worth changing at this point)